.idea/deploymentTargetDropDown.xml

```diff
@@ -3,7 +3,20 @@
   <component name="deploymentTargetDropDown">
     <value>
       <entry key="app">
-        <State />
+        <State>
```

Comment:
All .idea files should be ignored and not pushed to the repository. Please check if you have .gitignore in order to ignore idea files and other files that are not needed. If not find how to setup .gitignore file.

Reply:
I do have the .gitignore file in my project